### ISSUE TITLE
Cannot read property 'substring' of undefined

### ISSUE THREAD

**Issue by maxild** [VSCodeVim/Vim#7525]
**To Reproduce**

1.  Go to '...'
2.  Click on '....'
3.  Scroll down to '....'
4.  See error

**VSCodeVim version**: 1.22.1

<details><summary>Stack trace</summary>

```
TypeError: Cannot read property 'substring' of undefined
    at _.display (/home/maxfire/.vscode/extensions/vscodevim.vim-1.22.1/src/cmd_line/commandLine.ts:402:89)
    at statusBarText (/home/maxfire/.vscode/extensions/vscodevim.vim-1.22.1/src/statusBar.ts:199:35)
    at Object.clear (/home/maxfire/.vscode/extensions/vscodevim.vim-1.22.1/src/statusBar.ts:105:17)
    at /home/maxfire/.vscode/extensions/vscodevim.vim-1.22.1/extensionBase.ts:354:23
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

</details>

**Comment by elazarcoh** [comment]
That should be fixed now in version `1.22.2`

**Comment by J-Fields** [comment]
Fixed with #7513
